L/libRDKit/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+using BinaryBuilder, Pkg
+
+julia_version = v"1.6.0"
+
+name = "libRDKit"
+version = v"2021.09.1pre"
+
+sources = [
+    GitSource("https://github.com/rdkit/rdkit.git", "af3bb3e78b24ed8d92211d9c047ddbcf5c04afc8"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/rdkit
+mkdir build
+cd build
+cmake \
+-DCMAKE_BUILD_TYPE=Release \
+-DRDK_INSTALL_INTREE=OFF \
+-DRDK_BUILD_INCHI_SUPPORT=ON \
+-DRDK_BUILD_PYTHON_WRAPPERS=OFF \
+-DRDK_BUILD_CFFI_LIB=ON \
+-DRDK_BUILD_FREETYPE_SUPPORT=ON \
+-DRDK_BUILD_CPP_TESTS=OFF \
+-RDK_BUILD_SLN_SUPPORT=OFF \
+-DCMAKE_INSTALL_PREFIX=${prefix} \
+-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+-DCMAKE_PREFIX_PATH=${prefix} \
+..
+make -j${nproc}
+make install
+"""
+
+platforms = [
+    Platform("x86_64", "linux"),
+    Platform("i686", "linux"),
+    Platform("aarch64", "linux"),
+    Platform("x86_64", "macos"),
+    # Platform("aarch64", "macos"),
```

Comment:
What's the problem with this platform?

Reply:
I see the following warnings when building for it:

```
┌ Warning: Dependency FreeType2_jll does not have a mapping for artifact FreeType2 for platform aarch64-apple-darwin20-libgfortran5-cxx11
└ @ BinaryBuilderBase ~/.julia/packages/BinaryBuilderBase/6vjIZ/src/Prefix.jl:496
┌ Warning: Dependency Eigen_jll does not have a mapping for artifact Eigen for platform aarch64-apple-darwin20-libgfortran5-cxx11
└ @ BinaryBuilderBase ~/.julia/packages/BinaryBuilderBase/6vjIZ/src/Prefix.jl:496
┌ Warning: Dependency boost_jll does not have a mapping for artifact boost for platform aarch64-apple-darwin20-libgfortran5-cxx11
└ @ BinaryBuilderBase ~/.julia/packages/BinaryBuilderBase/6vjIZ/src/Prefix.jl:496
┌ Warning: Dependency Bzip2_jll does not have a mapping for artifact Bzip2 for platform aarch64-apple-darwin20-libgfortran5-cxx11
└ @ BinaryBuilderBase ~/.julia/packages/BinaryBuilderBase/6vjIZ/src/Prefix.jl:496
```
and then it fails to find boost, so I assumed that it couldn't find the dependencies.

```
[12:10:21] CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:164 (message):
[12:10:21]   Could NOT find Boost (missing: Boost_INCLUDE_DIR) (Required is at least
[12:10:21]   version "1.58.0")
[12:10:21] Call Stack (most recent call first):
[12:10:21]   /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:445 (_FPHSA_FAILURE_MESSAGE)
[12:10:21]   /usr/share/cmake/Modules/FindBoost.cmake:2162 (find_package_handle_standard_args)
[12:10:21]   CMakeLists.txt:369 (find_package)
```

Reply:
Right, we need first to build all the dependencies for the new platforms.  Good that this package requires Julia v1.6, it'll make it easier to add new platforms when we'll have all the dependencies in place.